src/hypofuzz/corpus.py

```diff
@@ -133,6 +135,18 @@ def _check_invariants(self) -> None:
     def _fuzz_key(self) -> bytes:
         return self._key + b".fuzz"
 
+    @property
+    def singletons(self) -> int:
+        # Because _every_ arc hit is counted at least once, singletons are those arcs that have that base hit,
+        # and then _one_ more on top of it.
+        singletons = [item for item in self.overall_arc_counts.values() if 2 == item]
```

Comment:
Is this true? I believe we only store arcs that we have executed (via `CustomCollectionContext`), which means singletons should be those with a count of 1, right?

Reply:
Arcs outside of the package, like in python itself, are also counted, which (since they're only run once during a given run) will only ever have a count of 1.